### DISCUSSION TITLE
Print() function having only 1 argument

### DISCUSSION THREAD

**Discussion by gawarev** [carbon-language/carbon-lang#1969] [Language design]
The `Print()` function takes only 1 argument. It would be better if it had like most other languages have multiple arguments or at least the `fmt.Sprintf()` like function which we have in golang.

**Comment by jonmeow** [comment]
Print is really just a test implementation. Once we have variadic args and more complete string support we'll probably rewrite it.

**Reply by gawarev** [reply]
That will be cool
